src/core/2d_primitives.js

```diff
@@ -292,6 +292,10 @@ p5.prototype.point = function() {
  * @param {Number} y3 the y-coordinate of the third point
  * @param {Number} x4 the x-coordinate of the fourth point
  * @param {Number} y4 the y-coordinate of the fourth point
+ * @param {Number} z1 the z-coordinate of the first point
+ * @param {Number} z2 the z-coordinate of the second point
+ * @param {Number} z3 the z-coordinate of the third point
+ * @param {Number} z4 the z-coordinate of the fourth point
```

Comment:
there are two different function signatures here. so what you want to do is leave these four lines out, but add the text descriptions after the parameters listed in lines 315, 318, 321, and 324 below